### DISCUSSION TITLE
How to inline_query on (/) to list commands

### DISCUSSION THREAD

**Discussion by kenjibailly** [telegraf/telegraf#1375] [Q&A]
Description from Telegram:
"Show an additional (/) button in the input field in all chats with bots. Tapping it types a '/' and shows the list of commands."

Source:
https://core.telegram.org/bots

Example picture:
![image](https://user-images.githubusercontent.com/9883910/108632241-258aa580-746e-11eb-9463-22cff9fc95f9.png)

How would I do this?
I know I can do an inline_query, but that will be only of use when typing "@name_of_bot ", how could I create this on command "/" ?

**Comment by wojpawlik** [comment]
https://core.telegram.org/bots/api#setmycommands
